common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AccessCondition.scala

```diff
@@ -64,6 +66,6 @@ object AccessStatus {
       case status if status.startsWith("permission required") =>
         Right(AccessStatus.PermissionRequired)
       case status =>
-        Left(new UnknownAccessStatus(status))
+        Left(new UnknownAccessStatus(str))
```

Comment:
Assuming this needs to be `status`?

Reply:
No, I changed that intentionally. `status` is `str` lowercased but, this being an error that gets displayed, it's more useful if it prints the actual string that caused the error.

Reply:
Maybe a note?

Reply:
And assuming we can `_` `status` then too.

Reply:
Not sure i follow the last comment...Rather than a note I'd lean towards renaming the variables to something more meaningful `status` and especially `str` are awful names so that line is quite confusing